src/main/java/frc/robot/commands/intake/StartAndStopIntake.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.Constants;
+import frc.robot.subsystems.Deployer;
+import frc.robot.subsystems.Feeder;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class StartAndStopIntake extends Command{
+    private IntakeSubsystem intake;
+    private double time;
+    private Deployer deployer;
+
+    public StartAndStopIntake(IntakeSubsystem intake, Deployer deployer) {
+        this.intake = intake;
+        this.deployer = deployer;
+        addRequirements(intake);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        intake.stopMotors();
+    }
+
+    @Override
+    public void execute() {
+        if (deployer.isDeployerReverseLimitSwitchClosed()) {
```

Comment:
Why can't this `isDeployerReverseLimitSwitchClosed()` condition moved to `isFinished()`? Then you can just remove this condition, and only call `setMotorSpeed()` without the `stopMotors()` (since that's handled in `end()`).

Reply:
Fixed